src/Components/Web.JS/src/Platform/Mono/MonoDebugger.ts

```diff
@@ -26,6 +33,8 @@ export function attachDebuggerHotkey(resourceLoader: WebAssemblyResourceLoader)
         console.error('Cannot start debugging, because the application was not compiled with debugging enabled.');
       } else if (!currentBrowserIsChrome) {
         console.error('Currently, only Microsoft Edge (80+), or Google Chrome, are supported for debugging.');
+      } else if (!isDebugging) {
+        console.error('_blazor_debug query parameter must be set to enable debugging.');
```

Comment:
Could we print the url here? E.g. `To launch with debugging, use `${location.href}?blazor_debug=true`? I wouldn't put too much effort in making sure it does not have two query tokens

Reply:
Done!